assets/prefabs/structures/dungeon/dungeonTownBuilding/ThermalSprings.prefab

```diff
@@ -0,0 +1,428 @@
+{
+    "StructureTemplate": {
+        "type": "GooeysQuests:dungeonTownBuilding",
+        "spawnChance": 1000
+    },
+    "SpawnBlockRegions": {
+        "regionsToFill": [
+            { "blockType": "ChiselBlocks:woolLlama_blue", "region": { "min": [-12, 3, 1], "size": [1, 1, 1]}},
```

Comment:
This ST looks pretty good, but from a player perspective the actual foyer (starting room with the signs) is huge compared to really narrow pools on either side. Maybe pull in the sign walls and make the pools more spacious? If you could fit 1-2 blocks of space inside each pool room for walking and even benches with the freed up space I think that would make for a more attractive pool! :-)